beacon-chain/state/state-native/getters_validator.go

```diff
@@ -501,3 +501,24 @@ func (b *BeaconState) PendingBalanceToWithdraw(idx primitives.ValidatorIndex) (u
 	}
 	return sum, nil
 }
+
+func (b *BeaconState) HasPendingBalanceToWithdraw(idx primitives.ValidatorIndex) (bool, error) {
```

Comment:
Would be good to have a function description even though it's obvious what this one does